comp/core/workloadmeta/collectors/catalog/options.go

```diff
@@ -3,11 +3,6 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-// TODO: (components) Move remote-only to its own catalog, similar to how catalog-less works
```

Comment:
Could you also remove the similar comment in [catalog.go](https://github.com/DataDog/datadog-agent/blob/d9ab04e0b44a941e112cb3289b9fcbfda10bf634/comp/core/workloadmeta/collectors/catalog/catalog.go#L33)

Reply:
ah yes sorry I thought I had removed all of those